examples/bevy/slint_bevy_adapter.rs

```diff
@@ -0,0 +1,236 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: MIT
+
+use std::sync::Arc;
+
+use slint::wgpu_24::wgpu;
+
+use bevy::{
+    prelude::*,
+    render::{
+        extract_resource::{ExtractResource, ExtractResourcePlugin},
+        render_graph::{self, NodeRunError, RenderGraph, RenderGraphContext, RenderLabel},
+        renderer::RenderContext,
+        settings::RenderCreation,
+        RenderApp, RenderPlugin,
+    },
+};
+
+pub enum ControlMessage {
+    ReleaseFrontBufferTexture { texture: wgpu::Texture },
+    ResizeBuffers { width: u32, height: u32 },
+}
+
+pub async fn run_bevy_app_with_slint(
```

Comment:
Maybe this module and this function could be documented.

Do I understand correctly that user who want to use Slint with Bevy should copy this module in their application? Maybe the readme should state that as well.

Reply:
Yes, in principle that's the idea.  It'll need docs anyway for later when we make this a published crate.